tasks/arrays.yml

```diff
@@ -11,7 +11,7 @@
 # Creating raid arrays
 # We pass yes in order to accept any questions prompted for yes|no
 - name: arrays | Creating Array(s)
-  shell: "yes | mdadm --create /dev/{{ item.name }} --level={{ item.level }} --raid-devices={{ item.devices|count }} {{ item.devices| join (' ') }}"
+  shell: "yes | mdadm --create /dev/{{ item.name }} --level={{ item.level }} --chunk={{item.chunk_size|default(512)}} --raid-devices={{ item.devices|count }} {{ item.devices| join (' ') }}"
```

Comment:
>        -c, --chunk=
              Specify chunk size in kilobytes.  The default when
              creating an array is 512KB.  To ensure compatibility with
              earlier versions, the default when building an array with
              no persistent metadata is 64KB.  This is only meaningful
              for RAID0, RAID4, RAID5, RAID6, and RAID10.

              RAID4, RAID5, RAID6, and RAID10 require the chunk size to
              be a power of 2, with minimal chunk size being 4KB.

              A suffix of 'K', 'M', 'G' or 'T' can be given to indicate
              Kilobytes, Megabytes, Gigabytes or Terabytes respectively.

So there's a slight behaviour change here for arrays with no persistent metadata, whatever that means.

Reply:
oh god :facepalm: 

I'll see what I can do here ;-)

Reply:
@markgoddard Added the metadata flag as well with the default of the latest current version, this supports setting the chunk default to 512K.  So not providing these should keep the current behaviour intact.

Also updated the default chunk size to `512K` rather than `512` for clarity reasons.